src/helpers/date-utils.ts

```diff
@@ -0,0 +1,20 @@
+export class DateUtils {
+  second: number;
```

Comment:
this should be final variable, do not put on constructor,
and for consistency, should be like this

this.second = 1000;
this.minute = 60 * second;
this.hour = 60 * minute;
this.day = 24 * hour;
this.year = 365 * day;

Reply:
Sip, done ya..